cores/esp32/esp32-hal-ledc.c

```diff
@@ -115,6 +115,10 @@ static void _ledcSetupTimer(uint8_t chan, uint32_t div_num, uint8_t bit_num, boo
     uint8_t group=(chan/8), timer=((chan/2)%4);
     static bool tHasStarted = false;
     static uint16_t _activeChannels = 0;
+#if CONFIG_IDF_TARGET_ESP32S2
```

Comment:
is this true just for S2 or also for C3 (S3 maybe?)

Reply:
I tested it on C3 as well. It works fine on it given that C3 has not such bit, thus thre is no issue/fix to be done.

Reply:
S3 not tested.